rust/windows-client/src-tauri/src/client/auth.rs

```diff
@@ -7,6 +7,18 @@ use std::path::PathBuf;
 use subtle::ConstantTimeEq;
 use url::Url;
 
+// TODO: Put this behind a "CI tests only" flag so that
+// official CI builds, and default local builds won't get the mock
+#[cfg(target_os = "linux")]
+#[path = "auth/token_storage_mock.rs"]
+mod token_storage;
+
+#[cfg(not(target_os = "linux"))]
+#[path = "auth/token_storage_keyring.rs"]
+mod token_storage;
```

Comment:
You could use a "plain" `cfg` instead that one has to set via `RUSTC` flags (i.e. not a feature).

Reply:
And then put it in RUSTFLAGS when calling Cargo?

That might be perfect, so the default would be to use `keyring-rs` and it'll only be disabled for CI with that flag until I can get it working headless.